### PR TITLE
OD-518 [Fix] - Ensure all Rich Text fields are initialized

### PR DESCRIPTION
**Pull request by skrupnyk**
@sofiiakvasnevska

OD-518 https://weboo.atlassian.net/browse/OD-518

## Description
All Rich text fields are initialised into TinyMCE editors

## Screenshots/screencasts
https://storyxpress.co/video/kps36n3w56zum6z3x

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko @YaroslavOvdii

### CODE REVIEW

File: templates/components/wysiwyg.build.hbs [hunk 1]
```diff
@@ -4,7 +4,7 @@
   v-model.trim.lazy="value"
   ref="textarea"
   :name="name"
-  :id="name"
+  :id="tinymceId"
```
Comment (tonytlwu) [inline templates/components/wysiwyg.build.hbs:7]: This tells me that you're using the form component ID as the ID for the WYSIWYG field. This means that if you have 2 WYSIWYG fields in a single form, they would clash as they would have the same ID.

**Review by tonytlwu — CHANGES_REQUESTED** [review]
Rejected +1

File: templates/components/wysiwyg.build.hbs (hunk 1 above)
Comment (skrupnyk) [reply templates/components/wysiwyg.build.hbs:7]: In this case, a unique ID is generated and an ID is added to it by name, this ID is not conflicted, this was provided. This was the only solution that I have found, in order to 'Rich text' has to be initialised into different forms. If there are still options, I can revise and redo.

Comment (tonytlwu) [reply templates/components/wysiwyg.build.hbs:7]: This works 👍🏻

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: High
Affected: Rich text form fields in apps
Verify: Rich text fields are working correctly

Squash and merge: Use PR title

**Pull request merged**